<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataFunctionalityDialogBase</class>
 <widget class="QDialog" name="MetadataFunctionalityDialogBase">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>798</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metadata-DB-Linker</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>320</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="kleNo_label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>430</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kle-journal-number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kleNoEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>430</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>635</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Latest change:</string>
   </property>
  </widget>
  <widget class="QLabel" name="responsible_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Responsible center or employee:</string>
   </property>
  </widget>
  <widget class="QLabel" name="project_label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>600</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projects which contain the file:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="kleNoLookupBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search KLE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="responsibleEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>570</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>600</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>731</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="treeDock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>731</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QPushButton" name="saveRecordButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>660</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Gem</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>630</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM-yyyy HH:mm</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRecordButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>660</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Slet</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>350</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chosefile_label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>22</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vælg filen som skal have tilføjet/ændret metadata</string>
   </property>
  </widget>
  <widget class="QLabel" name="metadatainformations_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Metadatainformations</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>350</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description of file content:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Descriping name:</string>
   </property>
   <property name="buddy">
    <cstring>nameEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="kleSuggestions">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>460</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>kleNoEdit</tabstop>
  <tabstop>kleNoLookupBtn</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>responsibleEdit</tabstop>
  <tabstop>projectEdit</tabstop>
  <tabstop>deleteRecordButton</tabstop>
  <tabstop>saveRecordButton</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
